automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/RegistrationPage.scala

```diff
@@ -48,8 +48,6 @@ class RegistrationPage(implicit webDriver: WebDriver) extends AuthenticatedPage
     programLocationCountryInput.setText(country)
 
     registerButton.doClick()
-    await condition registrationCompleteMessageIsPresent()
-    await condition !registrationCompleteMessageIsPresent()
```

Comment:
I think these are flakey. The registration complete message is fleeting and line 51 might miss it. E.g. https://fc-jenkins.dsp-techops.broadinstitute.org/job/firecloud-ui-fiab-test-runner/822/

the wait failed but the screen shot shows the library page